examples/ffva/src/app_conf.h

```diff
@@ -15,10 +15,15 @@
 #define appconfAUDIOPIPELINE_PORT      7
 #define appconfI2S_OUTPUT_SLAVE_PORT   8
 
+#ifndef appconfINTENT_ENGINE_READY_SYNC_PORT
+#define appconfINTENT_ENGINE_READY_SYNC_PORT      18
+#endif /* appconfINTENT_ENGINE_READY_SYNC_PORT */
+
 /* Application tile specifiers */
 #include "platform/driver_instances.h"
+#define ASR_TILE_NO             FLASH_TILE_NO
 #define FS_TILE_NO              FLASH_TILE_NO
-#define AUDIO_PIPELINE_TILE_NO  MICARRAY_TILE_NO
```

Comment:
Aren't mic array tile no. and flash tile no. different? Have we changed the tile on which the pipeline runs? Also, is this needed for the UA build as well?

Reply:
AUDIO_PIPELINE_TILE_NO is only used in the ASR code, and it is actually the tile with the pipeline output.
@shuchitak, should I rename it to AUDIO_PIPELINE_OUTPUT_TILE_NO?
In the FFD examples the whole pipeline runs on one tile, in FFVA AEC is on tile 1 and the rest in tile 0.

Reply:
hey @lucianomartin, yeah, AUDIO_PIPELINE_OUTPUT_TILE_NO would be a better name